contracts/test/README.md

```diff
@@ -0,0 +1,46 @@
+# Contract Tests
+
```

Comment:
thanks for the readme :). When do you choose to write fuzz vs invariant vs echidna tests?